fixtures/app_with_passing_tests/package.json

```diff
@@ -0,0 +1,28 @@
+{
+  "name": "@gestaltjs/app-with-passing-tests",
+  "version": "0.1.0",
+  "private": false,
+  "description": "App with passing tests Gestalt Fixture",
+  "keywords": [
+    "gestalt",
+    "gestaltjs",
+    "web"
+  ],
+  "license": "MIT",
+  "main": "index.json",
+  "publishConfig": {
+    "access": "public"
+  },
+  "dependencies": {
+    "vitest": "^0.1.23"
+  },
+  "engine-strict": true,
+  "engines": {
+    "node": "^12 || ^14 || ^16"
```

Comment:
I learned today that syntax is not valid. It should be `>= 12`. I think we'll have to update it in all the `package.json` in the repo.